### PR TITLE
add types

### PR DESCRIPTION
**Pull request by stefanprobst**
this adds types to `xastscript`. note: adds `@types/xast` as new dependency.

### CODE REVIEW

File: types/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es6"],
```
Comment (wooorm) [inline types/tsconfig.json]: I think it's es2015 in this case? But probably doesn't matter.

**Review by wooorm — COMMENTED** [review]
Nice!

File: types/tsconfig.json (hunk 1 above)
Comment (stefanprobst) [reply types/tsconfig.json]: changed. (i think it's [just an alias](https://www.typescriptlang.org/tsconfig/#high-level-libraries))

**Review by ChristianMurphy — APPROVED** [review]
Thanks @stefanprobst!

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Sweet, [released](https://github.com/syntax-tree/xastscript/releases/tag/2.0.0)!

**Pull request merged**